kitsune/wiki/migrations/0003_auto_20150506_2316.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you either name the migration to explain what it does, add a module level docstring explaining what it does, or both? I think that is a good habit to  be in for the new migration system.